lib/floe/runner.rb

```diff
@@ -27,6 +27,13 @@ def for_resource(resource)
         scheme = resource.split("://").first
         resolve_scheme(scheme) || raise(ArgumentError, "Invalid resource scheme [#{scheme}]")
       end
+
+      def runners
+        @runners.each_value.map do |runner|
+          runner = runner.call if runner.kind_of?(Proc)
+          runner
+        end
+      end
```

Comment:
This is essentially `@runners.values` resolving the `procs`.

---

I feel supporting multiple schemes by a single runner is currently far fetched.
Ignoring that use case allows us to keep to our current simple runner registration process: `Floe::Runner.register_scheme`.

In the future, if we come up with a case where there are multiple schemes, we can make our registration process more complicated.